open-science-projects.md

```diff
@@ -34,8 +36,95 @@ We chose our projects in September 2020, then worked on them full time from Janu
 ## Advice for undergrads selecting their projects
 If you are an undergrad considering what direction to take your studies try to complete an undergraduate project in a field you are considering entering into, even if you have no prior experience. I was considering completing an MSc in bioinformatics and therefore picked a bioinformatics-based project. I found the experience of stepping outside of my comfort zone and into a field where I had little experience daunting but very valuable. The project provided a positive experience and a fantastic learning curve which confirmed my initial interest in the field and allowed me to gain confidence and competence within coding and software. 
 
-## My top takeaways 
+## Our top takeaways 
 -	Being able to access other researchers work helped speed up the project. For example, when trying to create transcriptome annotation files for a new example species, I was able to look at pre-existing code and use it as a starting point for learning which functions and packages could be used. This saved me from starting from scratch and helped guide me
 -	It is easier to learn when resources are available to refer back to
 -	Open-Science is like a group project. Except the world is your group and they won’t affect your grade. 
 -	Documenting your work helps you, aids others and helps them help you 
+
+# The PI perspective
+
+The ideal student research project benefits everyone involved: students learn and make their grades, while the research project moves forward and the research group learns from the students.
+Open science projects can approach this ideal, by treating students as collaborators throughout their projects, and having a clear structure set up for them to engage and contribute.
+The key opportunity is that doing open science takes time, and it is hard to find others willing to climb the "mountain of engagement" from learning about a project to participating, collaborating, and leading.
+Students whose research project intersects with a larger open science project have the time and incentive to engage.
+
+Open-source software, like riboviz, needs users! 
+Users test features, read documentation, find bugs, and engaged users suggest improvements.
+Project students volunteered to do the project, then they want to learn and to earn good grades, so they are highly motivated users.
```

Comment:
Could this sentence be phrased differently? 

`As project students volunteered to do the project, they have shown interest, want to learn and earn good grades, therefore they are highly motivated users` for example?

Reply:
What about:
> Project students are highly motivated users because they volunteered to do the project, they want to learn and to earn good grades.

Reply:
Edited the paragraph with this and other flow improvements.